intel_extension_for_transformers/neural_chat/tests/api/test_model.py

```diff
@@ -0,0 +1,78 @@
+
+from intel_extension_for_transformers.neural_chat.models.chatglm_model import ChatGlmModel
+from intel_extension_for_transformers.neural_chat.models.llama_model import LlamaModel
+from intel_extension_for_transformers.neural_chat.models.mpt_model import MptModel
+from intel_extension_for_transformers.neural_chat.models.neuralchat_model import NeuralChatModel
+import unittest
+
+class TestChatGlmModel(unittest.TestCase):
+    def setUp(self):
+        return super().setUp()
+
+    def tearDown(self) -> None:
+        return super().tearDown()
+
+    def test_match(self):
+        result = ChatGlmModel().match(model_path='/models/chatglm2-6b-int4')
+        self.assertTrue(result)
+
+    def test_get_default_conv_template(self):
+        text = r"Conversation(name='chatglm2', system_template='{system_message}', system_message='', roles=('问', '答'), messages=[], offset=0, sep_style=<SeparatorStyle.CHATGLM: 8>, sep='\n\n', sep2=None, stop_str=None, stop_token_ids=None)"
+        result = ChatGlmModel().get_default_conv_template(model_path='/models/chatglm2-6b-int4')

```

Comment:
@Liangyx2  @lvliang-intel  can we use a small LLM model? since it's just for functionality check.

Same for MPT and other models below.

Reply:
Thanks for your advice. These UT tests are to improve the code coverage, we do not load the models here.